wiki/osu!_File_Formats/Osu_(file_format)/en.md

```diff
@@ -194,26 +194,58 @@ Hit Circle Syntax:
 `x,y,time,type,hitSound,addition`
 `164,260,2434,1,0,0:0:0:0:`
 
+x (Integer) ranges from 0 to 512 (inclusive) and y (Integer) ranges from 0 to 384 (inclusive).
+
+time (Integer) is in milliseconds from the beginning of the song.
+
+type (Integer) is a bitmap:
+
+1 = Circle, 2 = Slider, 4 = New combo, 8 = Spinner. More significant bits can be used to choose the next combo color.
+
+hitSound (Integer) is a bitmap:
+
+2 = Whistle, 4 = Finish, 8 = Clap
+
+The only actual "types" are circles, sliders, and spinners. New combo can be OR'd in to extend the behavior of the hit circle.
+
+addition (Integer:Integer:Integer:Integer) is optional, and defaults to "0:0:0:0:". The first value corresponds to the sample set, and the second value corresponds to additions.
+
+0 = Auto, 1 = Normal, 2 = Soft, 3 = Drum
+
+The third value corresponds to the sample index, e.g. normal-3. The fourth value declares the volume of the sample.
+
 Slider Syntax:
 
-`x,y,time,type,hitSound,sliderType|curveX:curveY|...|repeat,pixelLength|edgeHitsound:edgeAddition,addition`
+`x,y,time,type,hitSound,sliderType|curvePoints,repeat,pixelLength,edgeHitsounds,edgeAdditions,addition`
 `424,96,66,2,0,B|380:120|332:96|332:96|304:124,1,130,2|0,0:0|0:0,0:0:0:0:`
 
+x, y, time, and type behave the same as described in Hit Circle Syntax.
+
+hitSound and addition apply to all circles of the slider.
```

Comment:
If I make a quick map and drop a slider, it shows as `108,136,2598,2,0,L|328:228,1,200`, changing it to clap changes all the sounds and it becomes `108,136,2598,2,8,L|328:228,1,200`. Is the behavior then that hitSound and addition apply to circles not defined by edgesound and edgeaddition?

e.g. 160,128,35,2,0,L|232:144,11,69.9999978637696,0|14|0|14,1:0|2:0|1:0|2:0,2:1:0:0: